base-template.sh

```diff
@@ -850,11 +902,11 @@ should_run_update() {
 #####################################################
 execute_update() {
     if is_single_repo; then
-        if sh -s -- --single <"$INSTALL_SCRIPT"; then
+        if DO_UPDATE_ONLY="yes" sh -s -- --single <"$INSTALL_SCRIPT"; then
```

Comment:
Why an environment variable, why not an argument, like the other args? (e.g. `--single --update-only`)